### PR TITLE
Extend installation timeout gracefully on aarch64

### PR DESCRIPTION
**Pull request by okurz**
aarch64 installations often timeout. The reason so far is unknown, bugs have
been filed:
https://bugzilla.suse.com/show_bug.cgi?id=982136
https://bugzilla.suse.com/show_bug.cgi?id=982137

This change catches the case when on aarch64 the initial installation timeout
of 2000 s is hit, records a soft failure with a reference to the
aforementioned (first) bug and continues for up to another 2000 s. If the
installation is still not complete, the test will then hard fail.  No change
to the timeout for other architectures has been done.

Verification test run: http://lord.arch/tests/238

### CODE REVIEW

File: tests/installation/install_and_reboot.pm [hunk 1]
```diff
@@ -26,9 +26,20 @@ sub run() {
         push(@tags, "DIALOG-packages-notifications");
         $timeout = 5500;    # upgrades are slower
     }
-    while (1) {
-        assert_screen \@tags, $timeout;
-
+    my $keep_trying = 1;
+    while ($keep_trying) {
+        # try gracefully on aarch64 because of boo#982136
+        if (get_var('ARCH') eq 'aarch64') {
```
Comment (coolo) [inline tests/installation/install_and_reboot.pm]: why is everyone so keen on reimplementing check_var?

Comment (okurz) [inline tests/installation/install_and_reboot.pm]: because it's so easy ;-) why is there a `check_var` when this comparison can be done with perl means?

Comment (okurz) [inline tests/installation/install_and_reboot.pm]: done

**Comment by okurz** [comment]
updated:
* use check_var instead of manual check

**Pull request merged**